accelerator/config.c

```diff
@@ -0,0 +1,52 @@
+#include "config.h"
+#include "utils/logger_helper.h"
+
+#define CONFIG_LOGGER_ID "config"
```

Comment:
Should it be something more explicit?

Reply:
I think it should, otherwise it will mess with cclient's log . How about "ta_config"?

Reply:
How about `TA`? The verbose messages are only meaningful for debugging and bug reporting.